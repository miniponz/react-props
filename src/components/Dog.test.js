import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('Color Component', () => {
  it('matches the color thing', () => {
    const wrapper = shallow(<Dog name="Ana Dogg" age={2}  weight="27 lbs" />);

    expect(wrapper).toMatchSnapshot();
  });
});
