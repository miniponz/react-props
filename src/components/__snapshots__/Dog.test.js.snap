// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Component matches the color thing 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Ana Dogg
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    2
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    27 lbs
  </dd>
</dl>
`;
